# ObjectDetect.srv
# Request
sensor_msgs/Image image
---
# Response
int32[] x1s
int32[] y1s
int32[] x2s
int32[] y2s
float64[] confs
string[] labels
string[] fnames
